modules/webapp/src/main/elm/Messages/Comp/SearchMenu.elm

```diff
@@ -88,10 +88,10 @@ de =
     , switchSearchModes = "Zwischen den Such-Modi wechseln"
     , contentSearch = "Volltextsuche…"
     , searchInNamesPlaceholder = "Suche in Titeln…"
-    , fulltextSearchInfo = "Volltextsuche im Dokumenteninhalt und in Notizen."
+    , fulltextSearchInfo = "Volltextsuche im Inhalt und in Notizen."
     , nameSearchInfo = "Sucht in Namen von Korrespondent/Betreffend, Dokumenten und Notizen."
     , tagCategoryTab = "Tag-Kategorien"
-    , chooseOrganization = "Wähle eine Organisation"
+    , chooseOrganization = "Wähle ein Unternehmen"
```

Comment:
This is tricky imho. Not all organizations are companies or enterprises etc. You might have universities, clubs, associations etc. wdyt?

Reply:
I've gone alphabetically through the files and the first one already had Unternehmen so I've changed the other ones.

Reply:
Oh this was a mistake then. Many thanks for changing it again!